jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -78,6 +81,9 @@
     private boolean _notifyRemoteAsyncErrors = true;
     private boolean _relativeRedirectAllowed;
 
+    private HostPort _serverAuthority;
+    private SocketAddress _localAddress;
```

Comment:
We need to either add these fields to the copy constructor OR well document that they are not copied by the copy constructor.

I'm probably happy with either approach... whatever works best in the XML and makes more sense for configuring http and https connectors.